src/amo/components/GetFirefoxButton/index.js

```diff
@@ -39,13 +42,23 @@ export type Props = {|
 type InternalProps = {|
   ...Props,
   _base64url?: typeof base64url,
+  _tracking: typeof tracking,
   i18n: I18nType,
   userAgentInfo: UserAgentInfoType,
 |};
 
 export const GetFirefoxButtonBase = (props: InternalProps) => {
   const { addon, buttonType, className, i18n, userAgentInfo } = props;
   const _base64url = props._base64url || base64url;
+  const _tracking = props._tracking || tracking;
+
+  const onButtonClick = () => {
+    _tracking.sendEvent({
+      action: GET_FIREFOX_BUTTON_CLICK_ACTION,
+      category: GET_FIREFOX_BUTTON_CLICK_CATEGORY,
+      label: addon ? addon.guid : '',
```

Comment:
From the issue: "Event Label: Add-on GUID if it's the main Download Firefox button on a listing page. Empty if it's the Download Firefox button in the site header.", which is why I used `''`.

That also explains why we might not have an add-on. It can be a button in the site header.

Reply:
Oh, oops. I thought we had a different button for the top header.